types/mongodb/index.d.ts

```diff
@@ -173,17 +245,22 @@ export interface ClientSession extends EventEmitter {
      * or entire transaction as needed (and when the error permits) to better ensure that
      * the transaction can complete successfully.
      *
-     * IMPORTANT: This method requires the user to return a {@link Promise}, all lambdas that do not
-     * return a {@link Promise} will result in undefined behavior.
```

Comment:
Removed all @links that looked like this since they don't properly link anything in VSCode (not sure if they do on other editors)